### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by htc16** [opentibiabr/otservbr-global-archived#1148]
Meu server está caindo desde ontem que atualizei a src com as ultimas informações lançadas no repo, no caso, tenho os seguintes logs:

```
Thread 2 "tfs" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7ffff1d1e700 (LWP 556)]
strlen () at ../sysdeps/x86_64/strlen.S:106
```

**Esse log foi as ultimas funções executadas antes de cair.**
```
2020-05-06 11:28:39.403 ( 162.179s) [        F1D1E700]            actions.cpp:296   INFO| Player[268435670] canUse from position:x[65535], y[0], z[0]?
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:314   INFO| Player[268435670] canUse from his position!
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:322   INFO| Player[268435670] canUse cid[3161] from position:x[32934], y[32676], z[6]?
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:372   INFO| Get Item cid:[3161] Action
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:390   INFO| Regular Item
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:397   INFO| Item is a rune
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:774   INFO| Player[268435670] can execute action position:x[32934], y[32676], z[6]?
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:337   INFO| Creature[268435670] canUseFar to position:x[32934], y[32676], z[6],checkLineOfSight[1], checkFloor[1]
2020-05-06 11:28:39.404 ( 162.179s) [        F1D1E700]            actions.cpp:366   INFO| Creature can use far!
2020-05-06 11:28:39.594 ( 162.370s) [        F1D1E700]            actions.cpp:296   INFO| Player[268435670] canUse from position:x[32935], y[32677], z[6]?
106	../sysdeps/x86_64/strlen.S: No such file or directory.
date       time         ( uptime  ) [ thread name/id ]                   file:line     v| 
2020-05-06 11:28:40.946 (   0.002s) [main thread     ]             loguru.cpp:610   INFO| arguments: /home/server/tfs
2020-05-06 11:28:40.946 (   0.002s) [main thread     ]             loguru.cpp:613   INFO| Current dir: /home/server
2020-05-06 11:28:40.946 (   0.002s) [main thread     ]             loguru.cpp:615   INFO| stderr verbosity: 0
2020-05-06 11:28:40.946 (   0.002s) [main thread     ]             loguru.cpp:616   INFO| 
-----------------------------------
```

**Comment by rioma1990** [comment]
`2020-05-06 14:42:28.071 ( 238.550s) [        79D72700]            account.cpp:29    INFO| Getting account[2] coin balance.
2020-05-06 14:42:28.071 ( 238.551s) [        79D72700]            account.cpp:33    INFO| Query:[SELECT `coins` FROM `accounts` WHERE `id` = 2].
2020-05-06 14:42:28.072 ( 238.551s) [        79D72700]            account.cpp:41    INFO| Account:[2] Coin Balance:[0].
CM Test has logged in.
2020-05-06 14:51:20.456 ( 770.936s) [        79D72700]            actions.cpp:296   INFO| Player[268435458] canUse from position:x[65535], y[3], z[0]?
2020-05-06 14:51:20.456 ( 770.936s) [        79D72700]            actions.cpp:314   INFO| Player[268435458] canUse from his position!
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:556   INFO| Use Item Player[268435458], Position:x[65535], y[3], z[0], Index:[0], Item cid:[3253], isHotkey:[0]
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:406   INFO| Internal Use Item Player[268435458], Position:x[65535], y[3], z[0], Index:[0], Item cid:[3253], isHotkey:[0]
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:372   INFO| Get Item cid:[3253] Action
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:390   INFO| Regular Item
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:397   INFO| Item is a rune
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:453   INFO| Container
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:524   INFO| Add Container
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:528   INFO| Player internalUseItem Success!
2020-05-06 14:51:20.457 ( 770.936s) [        79D72700]            actions.cpp:572   INFO| Success using item!
2020-05-06 14:53:18.758 ( 889.238s) [        79D72700]            actions.cpp:296   INFO| Player[268435458] canUse from position:x[32350], y[32218], z[6]?

Loguru caught a signal: SIGSEGV
Stack trace:
10      0x7f487dba288f clone + 63
9       0x7f487c0486db /lib/x86_64-linux-gnu/libpthread.so.0(+0x76db) [0x7f487c0486db]
8       0x7f487bd756df /usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0xbd6df) [0x7f487bd756df]
7       0x55b183d4d8a3 ./otbr(+0x1f48a3) [0x55b183d4d8a3]
6       0x55b183cf7b22 ./otbr(+0x19eb22) [0x55b183cf7b22]
5       0x55b183d37c01 ./otbr(+0x1dec01) [0x55b183d37c01]
4       0x55b183c008fc ./otbr(+0xa78fc) [0x55b183c008fc]
3       0x55b183c007f7 ./otbr(+0xa77f7) [0x55b183c007f7]
2       0x7f487dbb5411 __vasprintf_chk + 193
1       0x7f487dade4d3 _IO_vfprintf + 8515
0       0x7f487dc0f5a1 /lib/x86_64-linux-gnu/libc.so.6(+0x18e5a1) [0x7f487dc0f5a1]
2020-05-06 14:53:18.759 ( 889.238s) [        79D72700]                       :0     FATL| Signal: SIGSEGV`

**Comment by ghost** [comment]
![test](https://user-images.githubusercontent.com/64913328/81194177-9d114600-8fbc-11ea-88ec-32c4d2aa2d7f.PNG)

**Comment by gpedro** [comment]
@rioma1990 how to reproduce this crash?

**Comment by Bipbop69** [comment]
> @rioma1990 how to reproduce this crash?

just click on something far / close 
@gpedro

**Comment by gpedro** [comment]
ok. testing

**Comment by gpedro** [comment]
can you test this fix #1149?

**Comment by Bipbop69** [comment]
fixed
